Documentation/templates/xcassets/swift3.md

```diff
@@ -26,6 +26,8 @@ You can customize some elements of this template by overriding the following par
 | `imageTypeName` | `ImageAsset` | Allows you to change the name of the struct type representing an image. |
 | `noAllValues` | N/A | Setting this parameter will disable generation of the `allColors` and `allImages` constants. |
 | `publicAccess` | N/A | If set, the generated constants will be marked as `public`. Otherwise, they'll be declared `internal`. |
+| `forceProvidesNamespaces` | true | If set to true, generates namespaces even for non namespacing asset folders (i.e. "Provides Namespace" is unchecked) |
```

Comment:
Do we want to change the default to `false` while we're adding this functionality?

Reply:
Good question. Changing it to `false` would be a breaking change and might alter users generated code. However it would be more logical :)

Reply:
I know, and we are heading towards a breaking release (6.0), so this might be the time to correct the behaviour. @AliSoftware thoughts?

Reply:
Since the next release will be a breaking one anyway, I'm all for fixing this now and set it to false by default 👍
Especially since thanks to this PR people migrating to 6.0 would still be able to restore the old behavior with this param 😉

Reply:
Side note: not sure if we started a migration guide for 6.0? (if not could be a nice occasion)